terraform/eks-cluster/commands/destroy.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
```

Comment:
nit: might want `set -e` here

Reply:
Intentionally left it off here because when you're deleting a bunch of these things - it's common that one of the delete steps fails because that thing has already been deleted for whatever reason. i.e. if you're trying to delete all, but the app has already been deleted, you want it to keep going